pkg/jtracer/jtracer.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2023 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jtracer
+
+import "github.com/opentracing/opentracing-go"
+
+type JTracer struct {
+	OT opentracing.Tracer
+}
+
+func OT(t opentracing.Tracer) JTracer {
```

Comment:
```suggestion
func NewJTracer(t opentracing.Tracer) JTracer {
```

Reply:
I think OT is better name. How would NewJTracer work when you need to create it with OTEL TracerProvider?

Reply:
Good point. Theoretically when we use OtBridge it is supposed to interoperable with OTEL instrumentation, such that both instrumentation styles can coexist for a time. I never tried it.